src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -161,8 +168,9 @@ void set_owner_for(mp::SSHSession& session, const std::string& root, const std::
     mp::utils::trim_end(vm_user);
     mp::utils::trim_end(vm_group);
 
-    run_cmd(session, fmt::format("sudo /bin/bash -c 'cd \"{}\" && chown -R {}:{} {}'", root, vm_user, vm_group,
-                                 relative_target.substr(0, relative_target.find_first_of('/'))));
+    if (!relative_target.empty())
+        run_cmd(session, fmt::format("sudo /bin/bash -c 'cd \"{}\" && chown -R {}:{} {}'", root, vm_user, vm_group,
+                                     relative_target.substr(0, relative_target.find_first_of('/'))));
```

Comment:
I have a feeling calling this with an empty `relative_target` should be an error, not a NOOP…

Reply:
You are right, and it is this way because of the tests. Again, we can remove it here and add some lines in the tests.

Reply:
I would rather, yes - and if anything, we should assert here it's not empty…

Reply:
Right, assertion seems to be the correct thing to do.